taipy/core/_entity/_entity_ids.py

```diff
@@ -11,16 +11,20 @@
 
 from __future__ import annotations
 
+from typing import Set
```

Comment:
```suggestion
from typing import TYPE_CHECKING, Set

if TYPE_CHECKING:
    from taipy import CycleId, DataNodeId, JobId, ScenarioId, SequenceId, SubmissionId, TaskId
```

I would do this to avoid the local import